service/ci/publish_to_docker.sh

```diff
@@ -2,8 +2,9 @@
 export $(egrep -v '^#' .env | xargs)
 export WEB_REGISTRY_URI=boptest_service-web
 export WORKER_REGISTRY_URI=boptest_service-worker
-export DOCKER_HUB_WEB_REGISTRY_URI=nrel/boptest-web
-export DOCKER_HUB_WORKER_REGISTRY_URI=nrel/boptest-worker
+export DOCKER_ORG=nrel
```

Comment:
This is just a question.  Ok, so the actual docker images are hosted under NREL's docker hub account?

Reply:
Indeed, but we are now disabling the docker registry and using Github packages for hosting images.